src/page-modules/assistant/layout.tsx

```diff
@@ -24,13 +28,17 @@ export type AssistantLayoutProps = PropsWithChildren<{
   initialFromFeature?: GeocoderFeature;
   initialToFeature?: GeocoderFeature;
   initialTransportModesFilter?: TransportModeFilterOption[] | null;
+  departureMode?: DepartureMode;
```

Comment:
```suggestion
  departureMode: DepartureMode;
```

Reply:
This is needed to please TypeScript. Either this or rearrange the types. We might want to do the latter here. The `{ empty: true }` type is unioned with the content type, but when we also have props in the layout it gets difficult. Should probably move the `{ empty: true }` type to be unioned with both the type for content and layout, i.e. `(AssistantContentProps & AssistantLayoutProps) | { empty: true }`. Thoughts?

Reply:
Ah! I see. I'm not really sure what is the best option here. I kind of feel like the `{ empty: true }` is more correct to be unioned with the content type, as the content can be empty and the layout still has props 🤔 I might be wrong here and that your suggestion makes more sense.

Reply:
Might want to keep `departureMode` optional and look at rearranging the types in a separate issue?

Reply:
Yes. I think we should have a session regarding types in their entirety. Have gotten confused with graphql types, duplicated types etcetc.

Reply:
Yep, it's a bit confusing. Let's have a session later and merge this for now. I think it looks good otherwise. This is also needed to complete https://github.com/AtB-AS/planner-web/pull/50